### ISSUE TITLE
Support for 4.41" Panel

### ISSUE THREAD

**Issue by GadgetUK** [repaper/gratis#13]
Is support for the 4.41" panel http://www.pervasivedisplays.com/products/441 already available ?
If not, what changes need to be implemented ?

**Comment by hrishi921** [comment]
I am working with a 4.41" display as well. I am through with the hardware connections. I was looking for libraries that support the display of images on the 4.41" screen. Is any support available at this point of time ?

**Comment by shawaj** [comment]
Pretty sure that is is closed source for anything above the 2.71" at the moment...

Although I am not an official source

**Comment by hrishi921** [comment]
I agree to that. Anything above the 2.7" right now is not friendly for
quick applications unless you have some development time that you can spend
understanding the data sheets in order to build a library.

**Comment by aslakr** [comment]
Does this also apply to the new (?) 4.2" http://www.pervasivedisplays.com/products/420 and http://www.pervasivedisplays.com/kits/ext2_kit ?

**Comment by shawaj** [comment]
The 4.2" works with the ext 2 kit but it has an internal timing circuit
(iTC) and so currently is more limited than the screens with external
timing circuit (eTC).

However the up side is that it's quicker and cheaper to build products with
the iTC and requires less on the software side too in principle.

It's a great size too!
On 30 Jun 2016 2:13 p.m., "Aslak Raanes" <notifications@github.com> wrote:

Does this also apply to the new (?) 4.2"
http://www.pervasivedisplays.com/products/420 and
http://www.pervasivedisplays.com/kits/ext2_kit ?

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/repaper/gratis/issues/13#issuecomment-229653902>, or mute
the thread
<https://github.com/notifications/unsubscribe/ADNCurgXLJLdNt8-6Vzk2IpnTi1aLhTBks5qQ8ENgaJpZM4B6wCa>
.

**Comment by repaper** [comment]
4.2" and EXT2 kit are now supported in [Pervasive ](http://www.pervasivedisplays.com) only.
The new driver IC ([iTC](http://www.pervasivedisplays.com/products/eTC_vs_iTC)) is totally different from the G2 of supported for sub 3" that open in this Git. Most of the time, you can update display follows the application note that can be downloaded at the panel webpage in Pervasive.
We are working on the new plan to support the new driver series of panels. You can ask for assistance to Pervasive if you need the sample code for the new panels before we renew the Repaper.org and Gratis Git.
Thanks.

**Comment by GadgetUK** [comment]
Could you provide an email address where we can get support for the new 4.2" iTC and ext2 kit please ?

**Comment by repaper** [comment]
You can submit your request to the [Technical Support](http://www.pervasivedisplays.com/contact/technical-support) webform.
